plugin/src/main/scala/tut/TutPlugin.scala

```diff
@@ -66,24 +73,20 @@ object TutPlugin extends AutoPlugin {
         val re    = tutNameFilter.value.pattern.toString
         tutOne(streams.value, r, in, out, cp, opts, pOpts, re)
       },
-      tutOnly <<= InputTask.createDyn(Def.setting((state: State) => Space ~> tutFilesParser(state))) {
-        Def.task{ in =>
-          Def.task{
-            val r     = (runner in Tut).value
-            val inR   = tutSourceDirectory.value // input root
-            val inDir = if (in.isDirectory) in
-            else in.getParentFile    // input dir
-            val outR  = tutTargetDirectory.value // output root
-            val out   = new File(outR, inR.toURI.relativize(inDir.toURI).getPath) // output dir
-            val cp    = (fullClasspath in Tut).value
-            val opts  = (scalacOptions in Tut).value
-            val pOpts = tutPluginJars.value.map(f => "–Xplugin:" + f.getAbsolutePath)
-            val re    = tutNameFilter.value.pattern.toString
-            tutOne(streams.value, r, in, out, cp, opts, pOpts, re)
-          }
-        }
+      tutOnly := {
+        val in = tutFilesParser(true).parsed
```

Comment:
The `true`/`false` parameter looks fishy (could instead be `Space ~> tutFilesParser`). Why does it require the `Space` in the first place?